simulator/simple_planning_simulator/test/test_simple_planning_simulator.cpp

```diff
@@ -135,32 +99,13 @@ void resetInitialpose(rclcpp::Node::SharedPtr sim_node, std::shared_ptr<PubSubNo
 }
 
 void sendGear(
```

Comment:
FYI;

When using `launch_testing`, you can write the following.
Since `sim_node` is declared in `generate_test_description()` and managed by `launch_testing`, there is no need to use `spin_once()`.

```python
import time
import rclpy
from autoware_auto_vehicle_msgs.msg import GearCommand


def sendGear(gear, pub_sub_node: rclpy.Node):
    cmd = GearCommand()
    cmd.stamp = pub_sub_node.get_clock().now().to_msg()
    cmd.command = gear
    for _ in range(10):
        pub_sub_node.pub_gear_cmd_.publish(cmd)
        rclpy.spin_once(pub_sub_node)
        time.sleep(0.01)
```